lib/virt_feature_test_base.pm

```diff
@@ -160,7 +160,12 @@ sub post_fail_hook {
     #(caller(0))[3] can help pass calling subroutine name into called subroutine
     $self->junit_log_provision((caller(0))[3]) if get_var("VIRT_AUTOTEST");
 
-    virt_utils::collect_host_and_guest_logs;
+    if (get_var('ENABLE_VIR_NET')) {
```

Comment:
ENABLE_VIR_NET is always enabled in a testsuite if the testsuite has such setting. I think you can use this if appropriate:
my ($package, $filename, $line) = caller;
if $package == xxxxx or $filename == yyyy

Reply:
yep, ENABLE_VIR_NET is always enabled in a testsuite if the testsuite has such setting.
had updated code already.